app.js

```diff
@@ -0,0 +1,73 @@
+const bookLists = [];
+// function to add a book
+function Book(title, author) {
+  this.title = title;
+  this.author = author;
+  bookLists.push({ title, author });
+}
+
+function UI() {}
+
+// Add book to page
+UI.prototype.addBookToPage = (book) => {
+  const list = document.getElementById('add-book');
+
+  // create p elements
+  const divEl = document.createElement('div');
+
+  divEl.innerHTML = `
+  <p class="title">${book.title}<p>
+  <p class="author">${book.author}<p>
+  <button class="delete">Remove</button>
+  <hr>
+  `;
+  list.appendChild(divEl);
+};
+
+// delete book
+UI.prototype.deleteBook = (target) => {
+  if (target.className === 'delete') {
+    target.parentElement.parentElement.remove();
+  }
+};
+
+// clear fields
+UI.prototype.clearFields = () => {
+  document.getElementById('title').value = '';
+  document.getElementById('author').value = '';
+};
+
+// event listener for add book
+document.getElementById('book-form').addEventListener('submit', (e) => {
+  // get form values
+  const title = document.getElementById('title').value;
+  const author = document.getElementById('author').value;
+
+  // instantiate a book
+  const book = new Book(title, author);
+
+  // instantiate UI
+  const ui = new UI();
+
+  // validate
+  if (title === '' || author === '') {
+    alert('Please fill in the fields');
+  } else {
+    // add book to the page
+    ui.addBookToPage(book);
+
+    // clear fields
+    ui.clearFields();
+  }
+  // local storage
+  localStorage.setItem('books', JSON.stringify(bookLists));
+  e.preventDefault();
+});
+
+//  event listner for delete
+document.getElementById('add-book').addEventListener('click', (e) => {
+  // instantiate UI
+  const ui = new UI();
+  ui.deleteBook(e.target);
+  e.preventDefault();
+});
```

Comment:
- [ ] Your localStorage is not working properly. You should create a function which should render all the books in the localStorage when the page will reload :blush:
- [ ] When we add books and then refresh the page then add the new books then the new books replace the old entries in the localStorage. Please fix this issue :+1:
- [ ] When we click on remove button then it doesn't remove the book from localStorage. Kindly fix it :thinking:

**Hint: Your code is not very much organized and you have a lot of changes to do so you can see this [file](https://github.com/HAMMAS-SALEEM/Awesome_Books/pull/1/files#diff-ed3ee7e0beea2498ff3b8ca85973d122fc6fa3d585d62b5807ec034d0cf076b3) to get an idea how you can complete this project in an expected way :+1:**